File: exercises/beecrowd/1019/1019.go
// https://judge.beecrowd.com/pt/problems/view/1019
package main

import (
	"fmt"
	"time"
)

func main() {
	var value int

	fmt.Scanln(&value)

	duration := time.Duration(value) * time.Second

	hours := int(duration.Hours())
	minutes := int(duration.Minutes()) % 60
	seconds := int(duration.Seconds()) % 60

	fmt.Printf("%d:%d:%d\n", hours, minutes, seconds)
}

File: exercises/beecrowd/1019/1019.java
// https://judge.beecrowd.com/pt/problems/view/1019
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int totalSeconds = sc.nextInt();

        int hours = totalSeconds / 3600;
        int minutes = (totalSeconds % 3600) / 60;
        int seconds = totalSeconds % 60;

        System.out.println(hours + ":" + minutes + ":" + seconds);

        sc.close();
    }
}
